homeUi/src/main/res/values/strings.xml

```diff
@@ -38,7 +38,7 @@
     <string name="gravatar_tab_avatar_upload_failure_image_too_big">The provided image exceeds the maximum size: 10MB</string>
     <string name="gravatar_tab_avatar_upload_error_action">Retry</string>
     <string name="gravatar_tab_avatar_upload_failure_dialog_remove_upload">Remove upload</string>
-    <string name="gravatar_tab_more_options_make_current">Make current</string>
+    <string name="gravatar_tab_more_options_make_current">Select</string>
```

Comment:
Because of the translation setup, it's always better to create a new String rather than modify an existing one. We already have the strings uploaded to GlotPress so we should follow that.

Reply:
Updated 👍